ynput/OpenPype
# AYON: Fix the bug on the limit group not being set correctly in Maya Deadline Setting

## Changelog Description
This PR is to bug-fix the limit groups from maya deadline settings errored out when the user tries to edit the setting.

## Additional info
n/a

## Testing notes:
1. Build server addon with this branch
2. Install the deadline addon into the AYON server
3. Make sure your deadline addon version is 0.1.7
4. Go to `ayon+settings://deadline/publish/MayaSubmitDeadline/limit/0`
![image](https://github.com/ynput/OpenPype/assets/64118225/0f9c98c0-cac7-49e3-8241-1660cd7e4580)

5. Add limit groups setting
6. Save Changes
7. It shouldn't error out anymore
8. Launch Maya via AYON launcher
9. Create Render Instance with renderable scene
10. Publish
11. It should be successfully rendered out and all the rendered images should be published to your publish folder after the rendering finished.
